packages/react/src/components/Accordion/AccordionTitle.tsx

```diff
@@ -55,24 +59,31 @@ class AccordionTitle extends UIComponent<ReactProps<AccordionTitleProps>, any> {
     index: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
     onClick: PropTypes.func,
     indicator: customPropTypes.itemShorthand,
+    icon: customPropTypes.itemShorthand,
   }
 
   handleClick = e => {
     _.invoke(this.props, 'onClick', e, this.props)
   }
 
   renderComponent({ ElementType, classes, unhandledProps, styles }) {
-    const { children, content, indicator, active } = this.props
+    const { children, content, icon, indicator, active } = this.props
     const indicatorWithDefaults = indicator === undefined ? {} : indicator
+    const indicatorIcon = icon === undefined ? {} : icon
 
     const contentElement = (
       <Layout
-        start={Indicator.create(indicatorWithDefaults, {
-          defaultProps: {
-            direction: active ? 'bottom' : 'end',
-            styles: styles.indicator,
-          },
-        })}
+        start={
+          <Layout
```

Comment:
please do not use `Layout` component as it's going to be deprecated and it renders a lot of extra DOM nodes; take a look at `Flex` component as it might solve your scenario

Reply:
What about creating a new example under Usages? If it's an example, then screenshot tests will capture any regressions - I don't think prototypes have that same benefit